internal/core/metadata/uom/bootstrap.go

```diff
@@ -10,12 +10,11 @@ import (
 	"os"
 	"sync"
 
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/container"
 	bootstrapContainer "github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/container"
 	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/startup"
 	"github.com/edgexfoundry/go-mod-bootstrap/v3/di"
 	"github.com/pelletier/go-toml"
```

Comment:
Since we are switching to yaml, do we still need `github.com/pelletier/go-toml` ?

Reply:
Yes, for now. Still used for some processing , which needs to be refactored  later.